src/getHumanAge.test.js

```diff
@@ -7,7 +7,23 @@ describe(`Function 'getHumanAge':`, () => {
     expect(getHumanAge).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should return element= 0 for catAge < 0`, () => {
+    expect(getHumanAge(-1, 24)).toEqual([0, 2]);
+  });
+
+  it(`should return element=0 for dogAge < 0`, () => {
+    expect(getHumanAge(15, -1)).toEqual([1, 0]);
+  });
+
+  it(`should return elements=0 for catAge, dogAge if input = 30 `, () => {
+    expect(getHumanAge(30, 30)).toEqual([0, 0]);
+  });
```

Comment:
but it's against the requirements!
if you mean that cats and dogs don't live for more than 30 years, in this task it means that if human age is 1, cat's age is 15, if human age is 2, cat's age is 24, etc.